### DISCUSSION TITLE
Evolution Documentation?

### DISCUSSION THREAD

**Discussion by bluecursor** [jacklandrin/OnlySwitch#87] [Q&A]
Can someone explain in the Evolution Section as there is really not enough documentation on it.

The Check Status Field: What triggers this and why? Where is the status shown?
Output Field:  What is the syntax for here? Where is it shown? What does it trigger?

Please explain.

**Comment by jacklandrin** [comment]
@bluecursor  Good question.
Switch status decides whether it is on or off. It will be checked when some operations happen, such as list appearing, trigger hotkey, etc. In the Evolution editor screen, the output will be shown in green when the debug button is pressed and the script can work out. If the output content is equal to true condition you input, the switch status will be on. 
I will update this information to README to let more people understand how to create evolution.

**Comment by evrpress** [comment]
Hi @jacklandrin!

Can you explain that a bit further, please?

No matter what I put in the textarea I always get a red cross:
<img width="1032" alt="image" src="https://github.com/user-attachments/assets/a594efe4-7567-4f04-a3cf-fcd74158f13c">

Is there some syntax or example available? 

Thanks!
